wg-node-lifecycle/charter.md

```diff
@@ -0,0 +1,102 @@
+# WG Node Lifecycle Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+There are many projects solving the node maintenance scenario and running into similar issues. The
+goal of this working group is to create unified APIs that the Kubernetes ecosystem can depend on.
+This will reduce the maintenance burden of these projects, and fix scenarios that block the node
+drain or cause improper pod termination. The goal is to have an easy to configure, out-of-the-box
+solutions that are compatible with the current APIs and behaviors.
+
+To properly solve the node drain, we must first understand the node lifecycle. This includes 
+provisioning/sunsetting of the nodes, PodDisruptionBudgets, API-initiated eviction and node
```

Comment:
>This includes provisioning/sunsetting of the nodes
>
I feel like this can be misunderstood.  We don't want to handle infrastructure provisioning.  We want to have a safe way to lifecycle the Node, so admins can work on the Node and end-user workloads are unaffected (besides capacity).